test/jdk/java/lang/Double/ParseDouble.java

```diff
@@ -557,10 +557,8 @@ private static void rudimentaryTest() {
     private static void testParsing(String [] input,
                                     boolean exceptionalInput) {
         for(int i = 0; i < input.length; i++) {
```

Comment:
As you're updating the file, I think changing the loop to an enhanced for loop would be an improvement.